docs/ocean/design_docs/prognostic-sediment-transport.rst

```diff
@@ -0,0 +1,476 @@
+and into suspension.
+To remain in suspension, the upward turbulence eddies must be greater than the sediment settling velocity.
+Suspended mass is exchanged vertically between the water column and the top bed layer.
+
+Suspended sediment transport can be solved either by emprical approaches or by a process-based approach.
+The emprical approaches solve SSC profile by the classic Rouse profile assumption.
+This approach requires the knowledge of the near-bottom reference SSC,
+which can be computed by one of the following two methods:
+
+- `Zyserman and Fredsoe, 1994 <https://ascelibrary.org/doi/pdf/10.1061/%28ASCE%290733-9429%281994%29120%3A9%281021%29>`_
+- `Goldstein et al., 2014 <https://esurf.copernicus.org/articles/2/67/2014/esurf-2-67-2014-discussion.html>`_
+
+The process-based approach solves the suspended sediment transport by the three-dimensional advection-diffusion equation
+(i.e., the scaler transport equation),  with an additional source/sink term for sediment vertical settling and exchange
+with the bed.
+
+Suspended sediment transport requires the knowledge of the sediment settling velocity.
+Sediment settling velocity is a function of the sediment geometry (e.g., particle size, shape and roundness),
+the fluid characteristics (density and viscosity), and well as the fluid turbulence velocity.
+Two sediment settling veloity formulations are selected to implement into MPAS-Ocean, including
+
+- `Soulsby, 1997 <https://www.icevirtuallibrary.com/doi/abs/10.1680/doms.25844.fm>`_
+- `Cheng, 1997 <https://doi.org/10.1061/(ASCE)0733-9429(1997)123:2(149)>`_
+
+
+Requirement: Morphological change
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/20
+
+Contributors: Zhendong Cao
+
+Morphological change here stands for the change of sea floor elevation (bathymetry) due to the convergence/divergence of net
+sediment flux during a time period. The Exner equation (`Paola and Voller, 2005 <https://doi.org/10.1029/2004JF000274>`_)
+is implemented to compute the morphological change.
+
+
+Algorithm Design
+---------------------------
+
+Algorithm Design: Sediment classes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/16
+
+Contributors: Zhendong Cao
+
+The number of sediment classes will be defined manually and considered as a dimension of sediment-related variables.
+
+Algorithm Design: Bedload sediment transport
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2021/01/16
+
+Contributors: Zhendong Cao
+
+In practice, a bedload coefficient should be defined to adjust the bedlad transport rate (see `COAWST code here
+<https://github.com/jcwarner-usgs/COAWST/blob/master/ROMS/Nonlinear/Sediment/sed_bedload.F#L608-L612>`_).
+
+The Meyer-Peter-Mueller and Soulsby-Dammgaard share the same expression of the bedload transport rate
+
+.. math::
+    q_b = \Phi \sqrt{(s-1)g D_{50}^3}\rho_s
```

Comment:
```suggestion
.. math::

    q_b = \Phi \sqrt{(s-1)g D_{50}^3}\rho_s
```

Reply:
Same in many other places.

Reply:
I will work through the document to fix them.